src/features/ITGovernance/Admin/Dates/index.tsx

```diff
@@ -1,277 +1,382 @@
+    <>
+      <PageHeading data-testid="grt-dates-view" className="margin-top-0">
+        {t('governanceReviewTeam:dates.heading')}
+      </PageHeading>
+      <h2>{t('governanceReviewTeam:dates.subheading')}</h2>
+      <Form onSubmit={handleSubmit(onSubmit)} className="maxw-mobile-lg">
+        <fieldset className="usa-fieldset margin-top-4">
+          <legend className="usa-label margin-bottom-1">
+            {t('governanceReviewTeam:dates.grtDate.label')}
+          </legend>
+          <HelpText id="TestDate-DateHelp-grt">
+            {t('governanceReviewTeam:dates.grtDate.format')}
+          </HelpText>
+          <div className="usa-memorable-date">
+            <div className="usa-form-group usa-form-group--month">
+              <Controller
+                name="grtDateMonth"
+                control={control}
+                render={({
+                  field: { ref, ...rest },
+                  fieldState: { error }
+                }) => (
+                  <FormGroup className="margin-top-2" error={!!error}>
+                    <Label className="text-normal" htmlFor="grtDateMonth">
+                      {t('general:date.month')}
+                    </Label>
+
+                    <TextInput
+                      {...rest}
+                      id="grtDateMonth"
+                      type="text"
+                      validationStatus={error && 'error'}
                     />
-                  );
-                })}
-              </ErrorAlert>
-            )}
-            {mutationResult.error && (
-              <ErrorAlert heading="Error">
-                <ErrorAlertMessage
-                  message={mutationResult.error.message}
-                  errorKey="systemIntake"
-                />
-              </ErrorAlert>
-            )}
-            <PageHeading data-testid="grt-dates-view" className="margin-top-0">
-              {t('governanceReviewTeam:dates.heading')}
-            </PageHeading>
-            <h2>{t('governanceReviewTeam:dates.subheading')}</h2>
-            <div className="tablet:grid-col-6">
-              <MandatoryFieldsAlert />
+                  </FormGroup>
+                )}
+              />
+            </div>
+            <div className="usa-form-group usa-form-group--day">
+              <Controller
+                name="grtDateDay"
+                control={control}
+                render={({
+                  field: { ref, ...rest },
+                  fieldState: { error }
+                }) => (
+                  <FormGroup className="margin-top-2" error={!!error}>
+                    <Label className="text-normal" htmlFor="grtDateDay">
+                      {t('general:date.day')}
+                    </Label>
+
+                    <TextInput
+                      {...rest}
+                      id="grtDateDay"
+                      type="text"
+                      validationStatus={error && 'error'}
+                    />
```

Comment:
React Hook Forms relies on refs for some functionality (scrolling to fields on error, for example). _Most_ of the form components we use have an `inputRef` prop that you can pass RHF's `ref` to. Can you update the fields in this form to use that prop instead of just leaving out the `ref`?